pkgs/development/compilers/tvm/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake, git }:
+stdenv.mkDerivation rec {
+  pname = "tvm";
+  version = "0.7.0";
+  src = fetchgit {
+    url = "https://github.com/apache/incubator-tvm.git";
+    rev = "v${version}";
+    sha256 = "0qflpd3lw0jslyk5lqpv2v42lkqs8mkvnn6i3fdms32iskdfk6p5";
+  };
+  nativeBuildInputs = [ cmake git ];
+  # TVM CMake build uses some sources in the project's ./src/target/opt/
+  # directory which errneously gets mangled by the eager `fixCmakeFiles`
+  # function in Nix's CMake setup-hook.sh to ./src/target/var/empty/,
+  # which then breaks the build. Toggling this flag instructs Nix to
+  # not mangle the legitimate use of the opt/ folder.
+  dontFixCmake = true;
+  enableParallelBuilding = true;
+
+  meta = with stdenv.lib; {
```

Comment:
If this works on all platforms, be sure to add an explicit `platforms = platforms.all;` so it is built by hydra.

Reply:
Unrelated to this PR: I have set this nowhere for my packages and they are all build on hydra for darwin and NixOS if I did not misinterpret that. To be honest I don't understand that flag fully.